nodejs/express-rabbitmq/app/src/app.ts

```diff
@@ -0,0 +1,62 @@
+import express from "express"
+import { setTag, setCustomData, expressErrorHandler } from "@appsignal/nodejs"
+import { trace } from "@opentelemetry/api"
+import cookieParser from "cookie-parser"
+import client from "amqplib";
+
+const port = process.env.PORT
+
+const app = express()
+app.use(express.urlencoded({ extended: true }))
+app.use(cookieParser())
+
+app.get("/", (_req: any, res: any) => {
+  res.send("200 OK")
```

Comment:
Would be good if this page displays a list of links the tester can click, so you don't need to open the code to figure out which routes work.

Reply:
@luismiramirez you said you were in the process of leaving on a comment on this?